source/javascripts/app/_copy.js

```diff
@@ -1,10 +1,24 @@
 function copyToClipboard(container) {
-  const el = document.createElement('textarea');
-  el.value = container.textContent.replace(/\n$/, '');
-  document.body.appendChild(el);
-  el.select();
-  document.execCommand('copy');
-  document.body.removeChild(el);
+  const text = container.textContent.replace(/\n$/, '');
+  
+  // Use the Clipboard API
+  navigator.clipboard.writeText(text)
+    .then(() => {
+    // Add success feedback
+      const button = container.parentNode.querySelector('.copy-clipboard');
```

Comment:
You should be able to pass the button into `copyToClipboard()` from the event handler avoiding an extra lookup.